src/lib/eslint/TypescriptEslintStrategy.ts

```diff
@@ -79,6 +78,47 @@ export class TypescriptEslintStrategy implements EslintStrategy {
 		return filteredPaths;
 	}
 
+	filterDisallowedRules(rulesByName: Map<string, ESRule>): Map<string, ESRule> {
+		// We want to identify rules that are deprecated, and rules that are extended by other rules, so we can manually
+		// filter them out.
+		const extendedRules: Set<string> = new Set();
+		const deprecatedRules: Set<string> = new Set();
+
+		for (const [name, rule] of rulesByName.entries()) {
+			if (rule.meta.deprecated) {
+				deprecatedRules.add(name);
+			}
+			if (rule.meta.docs.extendsBaseRule) {
+				// The `extendsBaseRule` property can be a string, representing the name of the rule being extended.
+				if (typeof rule.meta.docs.extendsBaseRule === 'string') {
+					extendedRules.add(rule.meta.docs.extendsBaseRule);
+				} else {
+					// Alternatively, it can just be the boolean true, which implies that the rule being extended shares
+					// the same name as this one, but is in the ESLint space instead of the Typescript space. So we strip
+					// the `@typescript-eslint/` from this rule's name and use that.
+					extendedRules.add(name.slice(19));
```

Comment:
This seems like it could fail if the eslint config doesn't follow the expected pattern. Something like this seems like it would be safer.
```
const prefix = '@typescript-eslint/';
if (name.indexOf(prefix) === 0) {
     extendedRules.add(name.slice(prefix.length));
} else {
     // Also log a warning
     extendedRules.add(name);
}
```

Reply:
Good point. We can make it even safer than that by looking for the last `/` character in the name and getting everything after that, since everything before that should be scoping. I'll make that change.